.github/workflows/go_test.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.19.x, 1.20.x]
+        go-version: [1.21.x]
```

Comment:
Add 1.22.x here?

Reply:
Agreed there should be two versions as per our recent conversations. Or 1.20.x if we don't want to support 1.22.x yet?

Reply:
done